doc/manuals/user/docker.md

```diff
@@ -1 +0,0 @@
-../../../docker/README.md
```

Comment:
Diff shows two opposite modifications on the same file but in different diff boxes... Weird, although I understand that without effect.

Reply:
NTC